docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,39 @@
+.. _config_http_filters_http_cache:
+
+Cache filter
+============
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.cache.v3.CacheConfig>`
+* :ref:`v3 SimpleHTTPCache API reference <envoy_v3_api_msg_extensions.cache.simple_http_cache.v3.SimpleHttpCacheConfig>`
+* This filter should be configured with the name *envoy.filters.http.cache*.
+* This filter doesn't support virtual host-specific configurations.
+
+The HTTP Cache filter implements most of the complexity of HTTP caching semantics.
+
+For HTTP Requests:
+
+* HTTP Cache respects request's `Cache-Control` directive. For example, if request comes with `Cache-Control: no-store` the request won't be cached.
+* HTTP Cache wont store HTTP HEAD Requests.
+
+For HTTP Responses:
+
+* HTTP Cache only caches responses with enough data to calculate freshness lifetime as per https://httpwg.org/specs/rfc7234.html#calculating.freshness.lifetime.
+* HTTP Cache respects `Cache-Control` directive from the upstream host. For example, if HTTP response returns status code 200 with `Cache-Control: max-age=60` and no `vary` header, it will be cached.
```

Comment:
```suggestion
* HTTP Cache respects ``Cache-Control`` directive from the upstream host. For example, if HTTP response returns status code 200 with ``Cache-Control: max-age=60`` and no ``vary`` header, it will be cached.
```